Python/emscripten_trampoline.c

```diff
@@ -78,6 +78,10 @@ EM_JS(CountArgsFunc, _PyEM_GetCountArgsPtr, (), {
 //     )
 // )
 addOnPreRun(() => {
+    let isIOS = /iPad|iPhone|iPod/.test(navigator.platform);
```

Comment:
Since `navigator` doesn't exist in node, we need:
```suggestion
    let isIOS = globalThis.navigator && /iPad|iPhone|iPod/.test(navigator.platform);
```